README.md

```diff
@@ -146,4 +146,4 @@ let myTooltip = new TenUp.tooltip( () => {
 
 ## Demo
 
-An example implementation can be found in the `index.html` file in the root of this package.
+An example implementation can be found in the `demo/index.html`.
```

Comment:
The phrasing of this sentence seems off. Maybe something like this:

Example implementations can be found in the `demo` directory.